application/fin/netsuite.go

```diff
@@ -5,24 +5,27 @@ import (
 	"bytes"
 	"fmt"
 	"log"
+	"time"
 
 	"github.com/silinternational/cover-api/api"
 	"github.com/silinternational/cover-api/domain"
 )
 
 const (
-	netSuiteHeader1 = `"RECTYPE","BATCHID", "BTCHENTRY","ORIGCOMP","SRCELEDGER","SRCETYPE","FSCSYR","FSCSPERD","SWEDIT","JRNLDESC","REVPERD","ERRBATCH","ERRENTRY","DETAILCNT","PROCESSCMD"` + "\n"
-	netSuiteHeader2 = `"RECTYPE","ACCTID","TRANSAMT","SCURNDEC","TRANSDESC","TRANSREF","TRANSDATE","SRCELDGR","SRCETYPE"` + "\n"
-
-	netSuiteSummaryRowTemplate     = `"1","000000","00001","","GL","JE","%d","%02d",0,"%s","00",0,0,0,2` + "\n"
-	netSuiteTransactionRowTemplate = `"2","%s",%s,"2","%s","%s",%s,"GL","JE"` + "\n"
+	netSuiteHeader                 = `"TRANSID","ACCTID","TRANSAMT","TRANSDESC","TRANSREF","TRANSDATE"` + "\n"
+	netSuiteTransactionRowTemplate = `%s,"%s",%s,"%s","%s",%s` + "\n"
 )
 
+var netsuiteRowNum int
+
 type NetSuite struct {
 	Period             int
 	Year               int
 	JournalDescription string
 	TransactionBlocks  TransactionBlocks
+
+	date   time.Time
+	annual int
```

Comment:
This feels like it should be a boolean. Otherwise, it is not clear what value it should hold.
```suggestion
	annual bool
```